src/energy_grid.F90

```diff
@@ -14,63 +14,17 @@ module energy_grid
 contains
```

Comment:
What do you think about a union grid which is comprised of all temperatures in that nuclide?  That won't really pay off until we really have lots of temperatures in the problem (and they aren't using an on-the-fly method like WMP) so it may be too early to be useful, but its a thought.